src/libraries/System.Threading.RateLimiting/src/System/Threading/RateLimiting/PartitionedRateLimiter.T.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Nit: why not just `PartitionedRateLimiter.cs` instead of `PartitionedRateLimiter.T.cs`.

Reply:
Because there will be a `static PartitionedRateLimiter` class for the `Create` and `CreateChained` methods which will go in `PartitionedRateLimiter.cs`.